src/test/run-make-fulldeps/coverage-reports/expected_show_coverage_counters.generics.txt

```diff
@@ -32,12 +32,12 @@ Combined regions:
   10:5 -> 12:6 (count=1)
 Segment at 10:5 (count = 1), RegionEntry
 Segment at 12:6 (count = 0), Skipped
-Emitting segments for function: _RNvXs_Cs4fqI2P2rA04_8genericsINtB4_8FireworklENtNtNtCs3rFBWs28XFJ_4core3ops4drop4Drop4dropB4_
+Emitting segments for function: _RNvXs_Cs4fqI2P2rA04_8genericsINtB4_8FireworklENtNtNtCs5T9nfiiJBPh_4core3ops4drop4Drop4dropB4_
```

Comment:
tbh, I have no idea where these changes are coming from…

Reply:
Probably the function name changed. This hash depends on quite a lot of things, if the name changed I wouldn't worry about it.

Reply:
Actually it's not a change, per se... The ordering has been swapped.

This is a problem with `llvm-cov` output. It generates output for multiple instantiations of generics in a non-deterministic order.

If the diff tests are going to fail because of this, we may need to make some changes. 

But I think this should have already been a problem, in which case maybe it's already handled.

I'll take another look at this later.

Reply:
This should be OK. As mentioned in another comment, the tests don't fail if counters files don't match, for multiple reasons.